### DISCUSSION TITLE
iotDb与flink对接问题咨询

### DISCUSSION THREAD

**Discussion by ArcDevil9** [apache/iotdb#3900] [Q&A] [answered]
iotDb可以直接对接flink，作为flink的数据源码？
我看flink-iotdb-connector，是flink写入iotdb，后面会考虑直接将iotdb作为flink的数据源的集成吗？
是推荐iotDb-kafka-flink这种组合吗，或者有更好的方案吗

**Comment by jixuan1989** [comment, marked as answer]
有一个 `IoTDBSource` 的实现类，但是很简单，可以传入一个SQL，从IoTDB中查询出来并推给Flink。
如果想做IoTDB中数据一旦更新，立即推送给Flink的话，要思考下是不是合适。。

**Reply by ArcDevil9** [reply]
好的  感谢
